spec/features/idv/in_person_spec.rb

```diff
@@ -107,8 +107,8 @@
       )
 
       # signing in again before completing in-person proofing at a post office
-      sign_in_and_2fa_user(user)
-      complete_doc_auth_steps_before_welcome_step
+      Capybara.reset_session!
+      sign_in_live_with_2fa(user)
       expect(page).to have_current_path(idv_in_person_ready_to_verify_path)
```

Comment:
IIRC they should only see the "ready to verify" screen if they're going through IdV again, such as through a partner request. Would that not be reset via `reset_session!` ? 🤔

Reply:
Even when signing in from the root sign-in page, not associated with a specific service provider? How do they manage account settings?

Reply:
This currently matches the behavior for password reset of GPO pending:

1. Users signing in for auth only get redirected to the SP
2. Users signing in for IdV get the ready to verify page
3. Users signing in without an SP get the ready to verify page

Reply:
This does seems like a good point though, @aduth. Unlike the password reset or GPO page, the ready to verify page does not include a link back to the profile.

Reply:
we created [LG-10119](https://cm-jira.usa.gov/browse/LG-10119) to address this!